src/app/test-results/test-results-list/test-results-detail-list.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ResponseTestResultDetail, TestResultService } from 'api-client';
+
+@Component({
+  selector: 'wh-test-results-detail-list',
+  templateUrl: './test-results-detail-list.component.html',
+})
+export class TestResultsDetailListComponent implements OnInit {
+  @Input() buildId: number;
+  @Input() summaryId: number;
+  details: ResponseTestResultDetail[];
+  failedDetails: ResponseTestResultDetail[];
+  private fetched: boolean;
+
+  constructor(
+    private testResultService: TestResultService) { }
+
+  ngOnInit(): void {
+    if (!this.fetched) {
```

Comment:
nit: Suggest inverting this if and return out early to reduce nesting.

```ts
ngOnInit(): void {
  if (!this.fetched) {
    // ...
  }
}
```

vs

```ts
ngOnInit(): void {
  if (this.fetched) {
    return;
  }
  // ...
}
```

If this function needs to do more in the future, then I suggest that in the future split the code into multiple functions that each return early inside themselves.

Reply:
Moving into its own function as a label on it feels nicer, and inverted the if-statement for early return 👍🏻